src/@utils/ocean.ts

```diff
@@ -4,22 +4,27 @@ import { AbiItem } from 'web3-utils/types'
 import Web3 from 'web3'
 
 export function getOceanConfig(network: string | number): Config {
-  const config = new ConfigHelper().getConfig(
-    network,
-    network === 'polygon' ||
-      network === 'moonbeamalpha' ||
-      network === 1287 ||
-      network === 'bsc' ||
-      network === 56 ||
-      network === 'gaiaxtestnet' ||
-      network === 2021000
-      ? undefined
-      : process.env.NEXT_PUBLIC_INFURA_PROJECT_ID
-  ) as Config
-  // TODO: remove hack once address is fixed
-  if (network === 'rinkeby' || network === 4)
-    config.oceanTokenAddress = '0x8967bcf84170c91b0d24d4302c2376283b0b3a07'
-  return config as Config
+  try {
+    const config = new ConfigHelper().getConfig(
+      network,
+      network === 'polygon' ||
+        network === 'moonbeamalpha' ||
+        network === 1287 ||
+        network === 'bsc' ||
+        network === 56 ||
+        network === 'gaiaxtestnet' ||
+        network === 2021000
+        ? undefined
+        : process.env.NEXT_PUBLIC_INFURA_PROJECT_ID
+    ) as Config
+    // TODO: remove hack once address is fixed
+    if (network === 'rinkeby' || network === 4)
+      config.oceanTokenAddress = '0x8967bcf84170c91b0d24d4302c2376283b0b3a07'
+    return config as Config
+  } catch (error) {
+    LoggerInstance.log('Error getting ocean config: ', error.message)
```

Comment:
```suggestion
    LoggerInstance.error('Error getting ocean config: ', error.message)
```

Reply:
or, that whole `try/catch` in here might be overkill as we are simple reading values from a json and if we don't find anything method just returns `null`. So almost sure the `try/catch` here would never be used